### PR TITLE
add community repo

### PR DESCRIPTION
**Pull request by usrbinkat**
UOR Requires a central hub for organizing community building resources, discussions, and projects.

This PR will allow for instantiating an organization level GH Discussions board amongst facilitating other community activity as well.

### CODE REVIEW

**Comment by afflom** [comment]
I concur with this change, but do not know how to validate this configuration.

**Review by afflom — APPROVED** [review]

**Comment by afflom** [comment]
@usrbinkat with the uor-community org now in use, can we close this?

**Comment by usrbinkat** [comment]
@afflom we still need this. We can box in the scope more narrowly but see [kubernetes/community](https://github.com/kubernetes/community) for examples of types of content we may place here that does not belong directly in uor-community

**Comment by jpower432** [comment]
@usrbinkat Any thoughts on converting the `emporous-docs` repo to community instead? I know that we are not 100% sure that the function of the repo is and it seems like the PRs against it are something that could go into the community repo. Thoughts?

**Comment by afflom** [comment]
@usrbinkat, Can you please rebase this and use the `previously`(https://github.com/uwu-tools/peribolos/blob/241265ba26e46bd1285f170b7a36c664364f56d8/README.md?plain=1#L43) flag to denote the emporous-docs to be changed to community? Please reach out if you have any questions.

File: peribolos.yaml [hunk 1]
```diff
@@ -53,9 +53,12 @@ orgs:
         description: Emporous client libraries and CLI implementation
         has_projects: true
         homepage: https://universalreference.io/
-      emporous-docs:
+      community:
         default_branch: main
+        description: Emporous Community
+        homepage: https://universalreference.io/community
```
Comment (sabre1041) [inline peribolos.yaml]: Can this be updated to refer to the `emporous.io` domain?

**Review by sabre1041 — CHANGES_REQUESTED** [review]

File: peribolos.yaml (hunk 1 above)
Comment (usrbinkat) [reply peribolos.yaml]: fixed

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**